ci/docker/image-builder/src/main/kotlin/ru/avito/image_builder/internal/docker/CliDocker.kt

```diff
@@ -0,0 +1,65 @@
+package ru.avito.image_builder.internal.docker
+
+import ru.avito.image_builder.internal.process.CommandExecutor
+import java.time.Duration
+
+internal open class CliDocker(
+    private val executor: CommandExecutor = CommandExecutor()
+) : Docker {
+
```

Comment:
Why not use any JVM client library?

I've tried to use https://github.com/docker-java/docker-java and found this approach overcomplicated because of many additional layers of abstraction. It takes too much time to satisfy their requirements and figure them out.
This is why I've decided to use CLI API. It's stable and simple enough.